### PR TITLE
Update pull req template for new line

### PR DESCRIPTION
**Pull request by lanseafood**
## Linked Issues:
| EPIC ⚡| [OCEANWATER-XXX](https://babelfish.arc.nasa.gov/jira/browse/OCEANWATER-XXX) |
) |
| :----------- | :----------- |
| Jira Ticket 🎟️   | [OCEANWATER-XXX](https://babelfish.arc.nasa.gov/jira/browse/OCEANWATER-XXX)) |
| Github :octocat:  | # |

## Summary of Changes
* 

## Test
*

### CODE REVIEW

**Comment by lanseafood** [comment]
template was missing new line

File: pull_request_template.md
Comment (Samahu) [inline pull_request_template.md:4]:
```suggestion
| EPIC ⚡ | [OCEANWATER-XXX](https://babelfish.arc.nasa.gov/jira/browse/OCEANWATER-XXX) |
| :----------- | :----------- |
| Jira Ticket 🎟️ | [OCEANWATER-XXX](https://babelfish.arc.nasa.gov/jira/browse/OCEANWATER-XXX) |
```

**Review by Samahu — CHANGES_REQUESTED** [review]
extra brackets need to be removed

**Review by Samahu — APPROVED** [review]

**Comment by kmdalal** [comment]
Three strong suggestions:

1. Change "Summary of Changes" to "Pull Request Summary" or "PR Summary".  This is only to make the automated email clear that this is a pull request -- there is no other explicit indicator.
2. Move "Linked Issues" to the bottom -- it is the least important of the three, sometimes N/A, and of no interest to external users.
3. Remove the link to babelfish (an internal Ames server), to reduce potential hack attacks.   I usually can't use the link, since I usually don't run VPN in Linux, and it's easy enough to find the Jiras without them.

**Comment by lanseafood** [comment]
> Three strong suggestions:
> 
> 1. Change "Summary of Changes" to "Pull Request Summary" or "PR Summary".  This is only to make the automated email clear that this is a pull request -- there is no other explicit indicator.
> 2. Move "Linked Issues" to the bottom -- it is the least important of the three, sometimes N/A, and of no interest to external users.
> 3. Remove the link to babelfish (an internal Ames server), to reduce potential hack attacks.   I usually can't use the link, since I usually don't run VPN in Linux, and it's easy enough to find the Jiras without them.

Hey Mike, some thoughts on these suggestions:

1. Isn't it implicit that this is a pull request since all of these are pull requests?
2. I think it's a pretty standard or template, so moving linked to the bottom seems more disorganizes and for PRs that do involve multiple ones, it is more useful. 
3. I do run VPN on my Linux so I can actually access the Jira links, and I think others do as well. I'm not sure about the potential hack risks but I find the links to be fairly useful, in particular if anyone wanted to cross check the PR with the tickets. We can remove if you feel very strongly about this though.

**Comment by kmdalal** [comment]
(1) Impacts only the _email_ generated by GitHub.  Without any template, the email states this is a PR.  With the template, all I see is the template, which doesn't mention "PR".    All I'm asking is to put the string "Pull Request" or "PR" into the template, at the top, so that the email is clearer.  That's my only strong request for this PR.

(2) I'm not familiar with PR template standards.  I guess I don't care where the Jira links are, I just want to see something saying this is a PR at the very top.

(3) I did a web search and find babelfish.arc.nasa.gov all over the place, so no problem leaving in the link :)

Another thing I notice is that some people don't fill out the template (especially the Linked Issues), and then this section is just a distraction.  It is printed in huge font at the top of the email.    I guess it's up to individuals to delete the template parts they aren't using -- that's better than leaving them blank.

**Comment by kmdalal** [comment]
I just created a PR (#153) for which the template didn't apply well, so I wrote over it.   I'm curious if the GitHub email you received (@Samahu and @lanseafood) explicitly tells you it is a PR, since I didn't use the template.

**Comment by kmdalal** [comment]
I took a closer look at my latest GitHub email about a new PR, and do see that "pull request" is mentioned if I scroll down.  Still would be nice to see it at the top.  Another way to do this and keep linked issues on top is to name that section something like "Linked Issues for this PR".

I also notice there's a second email from GitHub with just one sentence, saying my review is requested o this PR.

**Comment by Samahu** [comment]
> I just created a PR (#153) for which the template didn't apply well, so I wrote over it. I'm curious if the GitHub email you received (@Samahu and @lanseafood) explicitly tells you it is a PR, since I didn't use the template.

I can not tell based on the e-mail title if it is a PR, all I know that it is sent by github.

**Comment by Samahu** [comment]
I don't mean to intervene on the discussion but this PR was fixing a formatting issue that exist in the current active template. I would rather have this fixed and merged as soon as possible otherwise I have to remember fixing the formatting on my own for every new PR.

**Comment by lanseafood** [comment]
> I don't mean to intervene on the discussion but this PR was fixing a formatting issue that exist in the current active template. I would rather have this fixed and merged as soon as possible otherwise I have to remember fixing the formatting on my own for every new PR.

Yea I think some of the feedback is out of scope for what this PR was meant to address. I can't merge until there's 2 approvals though.

**Comment by kmdalal** [comment]
I'm sorry, I didn't realize until now that this is a hotfix -- the branch is not prefixed such and I didn't notice it was going to master.

Since my usability issues are mainly with how GitHub labels their email, I'll take it up with them, i.e. ask them to identify pull requests in the email subject line.

**Review by kmdalal — APPROVED** [review]

**Comment by Samahu** [comment]
> I'm sorry, I didn't realize until now that this is a hotfix -- the branch is not prefixed such and I didn't notice it was going to master.
> 
> Since my usability issues are mainly with how GitHub labels their email, I'll take it up with them, i.e. ask them to identify pull requests in the email subject line.

Not sure if this might help, but alternatively, you can create an Outlook e-mail rule to automatically sort GitHub e-mail into a separate folder. You can probably do more advanced stuff with it, but this is what I personally have configured:
image

**Comment by Samahu** [comment]
I merged the PR myself since @lanseafood is out and this fix is really needed. I don't wish to keep fixing the template on every new PR!

**Pull request merged**
